testData/deviceReporter/win32_installedSolutions.json

```diff
@@ -33,10 +37,6 @@
         },
 
         {
-            "id": "org.nvda-project"
-        },
-
-        {
             "id": "org.gnome.desktop.interface"
         },
 
```

Comment:
I'm not sure but I think this file is not required and should be removed. My understanding is the temporary Solutions registry done by these files is under development.